cluster-provision/cli/cli

```diff
@@ -190,6 +190,14 @@ if [ "$COMMAND" == "provision" ] ; then
   docker exec ${VM_CID} /bin/bash -c "scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i vagrant.key -P 22 /manifests/ceph/*.yaml vagrant@192.168.66.101:/tmp/ceph"
   docker exec ${VM_CID} /bin/bash -c "ssh.sh mkdir -p /tmp/cna"
   docker exec ${VM_CID} /bin/bash -c "scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i vagrant.key -P 22 /manifests/cna/*.yaml vagrant@192.168.66.101:/tmp/cna"
+  if [[ ${PREFIX} =~ "fedora" ]]; then
+    # Workaround for for vms that runs fedora31+ and docker,
+    # In order to work with docker on Fedora31 it is necessary to revert back to cgroups v1.
+    echo "Configuring cgroup version"
+    docker exec ${VM_CID} /bin/bash -c "ssh.sh sudo /bin/bash -s < /scripts/cgroups.sh &"
```

Comment:
is more elegant to use wait [1] you just get pid from proces and pass it to wait
https://www.gnu.org/software/bash/manual/html_node/Job-Control-Builtins.html#index-wait

also implement all that in the cgroups.sh script so it feels like a blocking script outside, doing conditionals with docker exec lines is super weird.

Reply:
Just to check if i got it,
do you mean remove the If clause, remove the while ssh_ready loop
and put wait before invoking cgroups.sh script?